### ISSUE TITLE
1.21+ Port

### ISSUE THREAD

**Issue by penturt** [XFactHD/BuddingCrystals#14]
Are there plans to port this mod over to 1.21+

**Comment by XFactHD** [comment]
Yes, a 1.21 port is being worked on but there are still some issues to work out.

**Comment by penturt** [comment]
Thank you for the quick response.

**Comment by penturt** [comment]
Just wanted to check on progress, if any...

**Comment by XFactHD** [comment]
There has unfortunately not been much progress but the aforementioned issues turned out to be a major pain to deal with and none of the solutions I have tried are satisfactory.

**Comment by penturt** [comment]
Would you mind if another mod dev worked on the port?

**Comment by XFactHD** [comment]
I would prefer to handle this myself. I have already been talking to other devs who have a lot of experience in the relevant areas about these issues, I have not been doing this alone.

**Comment by penturt** [comment]
I understand, thanks for your time. Guess I'll just check in every once a while.

**Comment by XFactHD** [comment]
The 1.21.1 port has been approved on CurseForge.

**Comment by penturt** [comment]
What! Omw to download.... Thank you.
